app/scripts/modules/ecs/src/serverGroup/configure/serverGroupConfiguration.service.ts

```diff
@@ -82,6 +85,7 @@ export interface IEcsServerGroupCommandBackingData extends IServerGroupCommandBa
   filtered: IEcsServerGroupCommandBackingDataFiltered;
   targetGroups: string[];
   ecsClusters: IEcsClusterDescriptor[];
+  capacityProviderDetails: IEcsCapacityProvidersDetails[];
```

Comment:
Inconsistency btwn the property name and the type (you should stick to either CapacityProviderDetails (IEcsCapacityProviderDetails) or CapacityProvidersDetails(IEcsCapacityProvidersDetails)

Reply:
Aahhh make sense.